.github/workflows/tag-release-version.yml

```diff
@@ -14,10 +14,17 @@ jobs:
       HAS_CHANGELOG: ${{ steps.ensure-changelog.outputs.HAS_CHANGELOG }}
     runs-on: ubuntu-latest
     steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
     - name: Ensure Changelog
       shell: bash
       run: |
-        changes=$(gh pr view ${{github.event.pull_request.number}} --json files -q '.files[].path')
+        git config --global user.name 'Automated Release'
+        git config --global user.email 'release-automation@bitmovin.com'
+        changes=$(git diff --name-only ${{github.event.pull_request.base.ref}})
```

Comment:
I guess this will not include files of some other PR that is merged in between current PR base ref?

Reply:
that's an interesting question - I think git only shows files as diffs when there is a change on the current branch on top of the base branch, and not the other way

Reply:
Got it.